content/03.methods.md

```diff
@@ -51,13 +51,78 @@ Numbers in parentheses show the number of positives and negatives that resulted
 
 ### Label Functions for Annotating Sentences
 
-The challenge of having too few ground truth annotations is common to many natural language processing settings, even when unannotated text is abundant.
+The challenge of having too few ground truth annotations is common too many natural language processing settings, even when unannotated text is abundant.
 Data programming circumvents this issue by quickly annotating large datasets by using multiple noisy signals emitted by label functions [@arxiv:1605.07723].
 Label functions are simple pythonic functions that emit: a positive label (1), a negative label (-1) or abstain from emitting a label (0).
 We combine these functions using a generative model to output a single annotation, which is a consensus probability score bounded between 0 (low chance of mentioning a relationship) and 1 (high chance of mentioning a relationship).
 We used these annotations to train a discriminator model that makes the final classification step.
 Our label functions fall into three categories: databases, text patterns and domain heuristics.
-We provide examples for each category in our [supplemental methods section](#label-function-categories).  
+
+#### Label Function Categories
+
+We provide examples for each category below. 
```

Comment:
Give me a 2-3 sentence intro to the idea of what a label function category is and list the broad classes (the things you bolded below).